### PR TITLE
fix config yamls

### PR DESCRIPTION
**Pull request by jpgomes**
This should fix `{"data_annotator_app.config.yaml":["[melanie.hanna@datarobot.com](mailto:melanie.hanna@datarobot.com)"],"dr_synth_data.config.yaml":["[alan.roberts@datarobot.com](mailto:alan.roberts@datarobot.com)"],"rssfeed.config.yaml":["[alan.roberts@datarobot.com](mailto:alan.roberts@datarobot.com)"]}
`

### CODE REVIEW

File: use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml [hunk 1]
```diff
@@ -7,8 +7,6 @@ languages:
   - python
 authors:
   - Mel Hanna
```
Comment (alevan10) [inline use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml:9]: is Mel still at this company? Could we switch this to someone else?

Comment (jpgomes) [reply use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml:9]: not anymore, can we leave her name as author? I will be the maintainer

Comment (alevan10) [reply use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml:9]: hm, let me check to see if that'll fail our test.

Comment (alevan10) [reply use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml:9]: it looks like those fields are optional, so we should be good

**Review by alevan10 — APPROVED** [review]

**Comment by jpgomes** [comment]
Thanks again @alevan10

**Pull request merged**